translations/kubectl/ko_KR/LC_MESSAGES/k8s.po

```diff
@@ -1,89 +1,2828 @@
-# Test translations for unit tests.
-# Copyright (C) 2017
-# This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR ianyrchoi@gmail.com, 2018.
-#
+# Hakyong DO <crazyraven@gmail.com>, 2018. #zanata
+# Ian Y. Choi <ianyrchoi@gmail.com>, 2018. #zanata
+# June Yi <gochist@gmail.com>, 2018. #zanata
 msgid ""
 msgstr ""
-"Project-Id-Version: gettext-go-examples-hello\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2013-12-12 20:03+0000\n"
-"PO-Revision-Date: 2018-04-03 06:05+0900\n"
-"Last-Translator: Ian Y. Choi <ianyrchoi@gmail.com>\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: EMAIL\n"
+"POT-Creation-Date: 2017-03-14 21:32-0700\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.0.6\n"
-"X-Poedit-SourceCharset: UTF-8\n"
-"Language-Team: \n"
-"Plural-Forms: nplurals=1; plural=0;\n"
-"Language: ko_KR\n"
+"PO-Revision-Date: 2018-08-08 06:17+0000\n"
+"Last-Translator: Ian Y. Choi <ianyrchoi@gmail.com>\n"
+"Language-Team: Korean\n"
+"Language: ko\n"
```

Comment:
I guess that this change from ```ko_KR``` to ```ko``` made the build failure.

Reply:
Done - thanks!

Reply:
It was just guessing. I don't know how to reproduce the automated verification in my local environment.